uk.co.compucorp.civicrm.hrcomments/xml/schema/CRM/HRComments/Comment.entityType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+// This file declares a new entity type. For more details, see "hook_civicrm_entityTypes" at:
+// http://wiki.civicrm.org/confluence/display/CRMDOC/Hook+Reference
+return array (
+  0 =>
+  array (
```

Comment:
you also need to add this to the extension's implementation of `hook_civicrm_entityTypes`